srcpkgs/libgdal/template

```diff
@@ -46,3 +82,17 @@ libgdal-devel_package() {
 		vmove usr/lib/*.so
 	}
 }
+
+python3-gdal_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - Python3 bindings"
+	pkg_install() {
+		vmove usr/bin/*.py
+		vmove "usr/lib/python*"
+		vlicense $wrksrc/$build_wrksrc/LICENSE.TXT
+		vdoc $wrksrc/$build_wrksrc/swig/python/README.rst
+		mkdir -p ${PKGDESTDIR}/usr/share/${pkgname}
+		cp -r $wrksrc/$build_wrksrc/swig/python/samples ${PKGDESTDIR}/usr/share/${pkgname}/examples
+		rm -rf ${PKGDESTDIR}/lib
```

Comment:
There should only be `/lib` if you would have created one in `pkg_install` but you didn't so this line should be a noop and be removed.